rfcs/1-jhipster-rfc-k8s-operator.md

```diff
@@ -147,9 +147,9 @@ The README.md file provides a glimpse of this first iteration, but after a lot o
 
 - **Q: This definitely is cool but is this really necessary for a JHipster application.**
 - **A**: If you are running in Kubernetes, the JHipster K8s Operator will provide you extra functionality and it will allow Kubernetes to understand JHipster specific concerns.
-- **Q: Are we rebuilding any features provided by tools like Istio(coz many of the proposed features looks very similar to what it does)**
+- **Q: Are we rebuilding any features provided by tools like Istio(coz many of the proposed features look very similar to what it does)**
```

Comment:
```suggestion
- **Q: Are we rebuilding any features provided by tools like Istio (because many of the proposed features look very similar to what it does)?**
```